libs/feature/admin/src/lib/user/delete-user.ts

```diff
@@ -0,0 +1,69 @@
+import { PrismaClient } from "@prisma/client";
+
+
+export async function deleteUserAndDependentData(username: string,
+     deletedEntities: DeletedEntities[],
+      database: PrismaClient) {
```

Comment:
muss das übergeben werden? database ist in prod eig als globale variable verfügbar? ist das wegen testability?

Reply:
das war ide idee dahinter